js/main.js

```diff
@@ -1400,6 +1399,15 @@ function initStationName() {
     }
 }
 
+function setDefaultValues() {

```

Comment:
bissl komischer Name, aber ist halt in Linie mit `setDefaultCurrentValues` 
würde aber ein `setDefaultGameDataValues()` bevorzugen.